### PR TITLE
Allow to use multiplte jobs modules

### PR DESCRIPTION
**Pull request by likeon**

### CODE REVIEW

**Comment by likeon** [comment]
I have fixed the issues.

Pushed with force, so reviews have been lost. Sorry about that.

**Comment by Koed00** [comment]
I've looked at the code and I would prefer it we just kept the  single `RQ_JOBS_MODULE` and give people the option to pass it a string or a tuple.

You can then check to convert it:

```
modules = settings.RQ_JOBS_MODULE
if type(modules) != tuple:
                    modules = (modules,)
```

This way you don't need another setttings constant.
What do you think?

**Comment by likeon** [comment]
Looks good.

Do you prefer to store changes squashed or should I make another commit?

**Comment by Koed00** [comment]
I have no preference, so do what you prefer.  You might need to check why your changes are failing on Python 3+ before we can merge.

**Comment by likeon** [comment]
My [pull-request](https://github.com/nvie/rq/pull/565) to rq which fixed the issue with Job.**eq** method was recently merged. Also I have fixed tests with the current rq version.

**Comment by Koed00** [comment]
Nice. That's great work. Let me go over the code.

File: README.md [hunk 1]
```diff
@@ -27,11 +27,14 @@ INSTALLED_APPS = (
     "django_rq_jobs",
 )
 ```
-* Add `RQ_JOBS_MODULE` in settings.py.
-   This will point RQ Jobs to where you keep your jobs. Anything marked with the  Django RQ's `@job` decorator 
+* Add `RQ_JOBS_MODULES` in settings.py.
```
Comment (Koed00) [inline README.md]: should be singular  'RQ_JOBS_MODULE'

File: README.md [hunk 2]
```diff
@@ -27,11 +27,14 @@ INSTALLED_APPS = (
...
+   A tuple of strings designating all modules where you keep your jobs. Anything marked with the  Django RQ's `@job` decorator 
    will show up in the admin.
 ```python
-RQ_JOBS_MODULE = 'myapp.tasks'
+RQ_JOBS_MODULES = (
```
Comment (Koed00) [inline README.md]: same thing

File: django_rq_jobs/models.py [hunk 3]
```diff
@@ -31,25 +31,38 @@ def queue_index_by_name(name):
 
 def task_list():
     """
-    Scans the module set in RQ_JOBS_MODULE for RQ jobs decorated with @task
+    Scans the modules set in RQ_JOBS_MODULES for RQ jobs decorated with @task
```
Comment (Koed00) [inline django_rq_jobs/models.py:35]: should be singular too. minor

**Comment by Koed00** [comment]
Sorry. I didn't merge this sooner. I didn't get an email about your last commit and I've been kind of busy with Django-Q.

**Pull request merged**